commands/partner_interconnect_attachments.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+// Partner creates the partner interconnect attachments command.
+func Partner() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "partner",
```

Comment:
I'm curious about `partner` as a top level command. Do we expect there to be other "partner" features or products in the future? As this is so closely related to VPCs, I wonder if it shouldn't live under there? Where will this functionality live in the control panel?